Dockerfile

```diff
@@ -49,7 +49,7 @@ RUN set -ex && \
       libpq5 zlib1g libyaml-0-2 libssl1.1 \
       libgdbm6 libreadline8 libncurses5 libffi7 \
       libxml2 libxslt1.1 libcurl4 libicu66 libre2-dev tzdata unzip libimage-exiftool-perl \
-      libcurl4-openssl-dev libexpat1-dev gettext libz-dev libssl-dev libpcre2-dev build-essential git-core \
+      libcurl4-openssl-dev libexpat1-dev gettext libz-dev libssl-dev libpcre2-dev build-essential \
```

Comment:
These packages (added in #2553) can be moved to BUILD_DEPENDENCIES as well. This can reduce the size of the image by 200MB.

Reply:
Thanks for this hint. Do you have any other comments or hints?